client/client.go

```diff
@@ -655,6 +655,11 @@ type Destination interface {
 //   * Metadata cannot be generated for the downloaded data
 //   * Generated metadata does not match local metadata for the given file
 func (c *Client) Download(name string, dest Destination) (err error) {
+	err, _ = c.DownloadWithCustom(name, dest)
+	return err
+}
+
+func (c *Client) DownloadWithCustom(name string, dest Destination) (err error, custom *json.RawMessage) {
```

Comment:
My understanding is that custom is downloaded anyway. Does it make sense to add a method to Client instead, say getCustomMetadata and people who need it call that after the download?